File: src/projects/Children/ChildMaterializedView/Domain/Entities/Question.cs
using Core.Persistance.Repositories.EntityFramework;

namespace Domain.Entities
{
    public class Question : Entity
    {
        public string _id { get; set; }
        public string QuestionTitleId { get; set; }
        public string QuestionName { get; set; }
        public virtual QuestionTitle QuestionTitle { get; set; }
        public virtual ICollection<QuestionAnswer> QuestionAnswers { get; set; }
        public Question()
        {

        }

        public Question(string id, string questionTitleId, string questionName):this()
        {
           _id= id;
            QuestionTitleId = questionTitleId;
            QuestionName = questionName;
        }
    }
}

File: src/projects/Children/question/src/main/java/com/krm3d/question/domain/entities/Question.java
package com.krm3d.question.domain.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import org.springframework.data.mongodb.core.mapping.Document;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Document
public class Question extends MongoDbEntity{

    private String questionTitleId;
    private String questionName;
}
